docs/griptape-tools/official-tools/file-manager.md

```diff
@@ -3,7 +3,63 @@
 This tool enables LLMs to save and load files.
 
 ```python
+from griptape.structures import Agent
 from griptape.tools import FileManager
 
-FileManager()
+# Set up an agent using the FileManager tool
+file_manager_tool = FileManager(dir='/path/to/files')
```

Comment:
@trish11953 `/path/to/files` will fail in the integration tests since it does not exist. Update the `dir` to `.` and use the `FileManager` to first create a file. Then have it read the file back to you.